moveit_ros/perception/semantic_world/CMakeLists.txt

```diff
@@ -12,6 +13,6 @@ ament_target_dependencies(moveit_semantic_world
   Eigen3
   Boost
 )
-target_link_libraries(moveit_semantic_world ${OpenCV_LIBRARIES})
+target_link_libraries(moveit_semantic_world PRIVATE moveit_compatibility)
```

Comment:
@JafarAbdi Why did you remove OpenCV_LIBRARIES?

Reply:
I thought it was already in the `ament_target_dependencies` should be added there anyway rather than using XXX_libraries

Reply:
That makes sense. I'll switch back to how you had it.